wavemaker-labs/ros2can_bridge
# Update Code to ROS2 Humble

Update Code to ROS2 Humble and add config and launch file

**Testing Procedure**
1. sudo apt install can-utils
2. sudo apt install net-tools
3. Setup Virtual CAN
```
  sudo slcan_attach -f -s4 -o /dev/tty4
  sudo slcand -S 115200 /dev/tty4 can0
  sudo ifconfig can0 txqueuelen 10000
  sudo ifconfig can0 up
```
4. ros2 topic echo /CAN/can0/receive
5. cansend can0 6b0#02F0010002004F4F

Expected Output:
```
header:
  stamp:
    sec: 0
    nanosec: 0
  frame_id: ''
id: 1712
is_rtr: false
is_extended: false
is_error: false
dlc: 8
data:
- 2
- 240
- 1
- 0
- 2
- 0
- 79
- 79
---
```
